webapp/packages/plugin-data-export/src/ExportNotification/ExportNotification.tsx

```diff
@@ -52,84 +42,63 @@ export const ExportNotification: React.FC<Props> = observer(function ExportNotif
-          </message>
-        </body-text-block>
-        <notification-footer as='div'>
-          <footer-time as='span'>{timeStringFromTimestamp}</footer-time>
-          <actions as="div">
-            {controller.isSuccess && (
-              <>
-                <Button
-                  type="button"
-                  mod={['outlined']}
-                  onClick={controller.delete}
-                >
-                  {translate('data_transfer_notification_delete')}
-                </Button>
-                <Button
-                  tag='a'
-                  href={controller.downloadUrl}
-                  mod={['unelevated']}
-                  download
-                  onClick={controller.download}
-                >
-                  {translate('data_transfer_notification_download')}
-                </Button>
-              </>
-            )}
-            {controller.hasDetails && (
+  return styled(useStyles(styles))(
+    <SnackbarWrapper closing={false} closeable={!controller.isPending} onClose={() => notification.close(false)}>
+      <SnackbarStatus status={controller.isPending ? ENotificationType.Loading : exportNotificationType} />
+      <SnackbarContent>
+        <SnackbarBody title={translate(controller.status)} message={controller.sourceName}>
+          {controller.task?.context.sourceName && (
+            <pre title={controller.task?.context.sourceName}>
+              {controller.task?.context.sourceName}
+            </pre>
+          )}
+        </SnackbarBody>
+        <SnackbarFooter timestamp={notification.timestamp}>
+          {controller.isSuccess && (
+            <>
               <Button
                 type="button"
                 mod={['outlined']}
-                disabled={controller.isDetailsDialogOpen}
-                onClick={controller.showDetails}
+                onClick={controller.delete}
               >
-                {translate('ui_errors_details')}
+                {translate('data_transfer_notification_delete')}
               </Button>
-            )}
-            {controller.isPending && (
               <Button
-                type="button"
-                mod={['outlined']}
-                disabled={controller.process?.getState() === EDeferredState.CANCELLING}
-                onClick={controller.cancel}
+                tag='a'
+                href={controller.downloadUrl}
+                mod={['unelevated']}
+                download
+                onClick={controller.download}
               >
-                {translate('ui_processing_cancel')}
+                {translate('data_transfer_notification_download')}
               </Button>
-            )}
-          </actions>
-        </notification-footer>
-      </notification-body>
-      {!controller.isPending && (
-        <IconButton name="cross" viewBox="0 0 16 16" onClick={controller.delete} />
-      )}
-    </notification>
+            </>
+          )}
+          {controller.hasDetails && (
```

Comment:
seems like we can replace `hasDetails`, `isPending`, `isSuccess` with notification type